src/App.tsx

```diff
@@ -1,9 +1,26 @@
-import React from 'react';
+import React, { useState } from 'react';
 import './App.scss';
 import { MoviesList } from './components/MoviesList';
 import moviesFromServer from './api/movies.json';
 
 export const App: React.FC = () => {
+  const [query, setQuery] = useState('');
+
+  function handleChange(event: React.ChangeEvent<HTMLInputElement>) {
+    setQuery(event.currentTarget.value);
+  }
+
+  const visibleMovies = moviesFromServer.filter(movie => {
+    const lowerCaseInput = query.trim().toLowerCase();
+    const lowerMovieTitle = movie.title.toLowerCase();
+    const lowerMovieDescription = movie.description.toLowerCase();
```

Comment:
implement filter by adding the ability to ignore extra spaces between entered words.
It would fun experience, i promise.